kolibri/core/assets/src/views/sync/ConfirmationRegisterModal.vue

```diff
@@ -90,6 +99,13 @@
             }
           });
       },
+      cancel() {
+        if (this.alreadyRegistered && this.successOnAlreadyRegistered) {
```

Comment:
If a facility tries to re-register to a project they are already registered to, should there be a check to see if it is already registered with the project before submission, which would change this data prop `alreadyRegistered` to true? Or is this line only meant to catch the error if the facility is found to already be registered?

I'm asking because the parts where `alreadyRegistered` is used in the template are never really shown and could be removed. I manipulated the data in Vue.js devtools to see this view.

<img width="964" alt="Screenshot 2023-05-25 at 3 46 48 PM" src="https://github.com/learningequality/kolibri/assets/46411498/2efd3de1-94e2-4f32-9f77-ceb235406f9e">

Reply:
This is only meant to catch the case where there is an error with registration due to the facility already being registered to KDP.

In the case where we are directly doing a registration action by using the `Register` option from the dot menu, this emits a 'cancel' event. In the case that I am using it here, I wanted it to instead emit a success event in this case, as it is fine to carry on and try to sync to KDP because it is already registered, so this logic is intended to be handling that.